script/input/1/integration.json

```diff
@@ -6,11 +6,13 @@
         },
         "optimism": {
             "rpc": "OPT_RPC_URL",
+            "block": "OPT_BLOCK",
```

Comment:
@iamchrissmith @hexonaut so IMO I think we should not have `rpc` or `block` here. We won't be changing this file as `dss-test` will be used as a dependency. Even if the file can be replaced placing a new one in the root project that is calling this dependency IMO seems confusing. This file should only have the addresses that we want to refer: "chainlogs and message passing contracts".

Reply:
Yeah I think we should standardize around how these environment variables are used. I've just been picking names as I go similar to the `ETH_RPC_URL`. I'm okay with removing this and just assuming something like `OPT_RPC_URL` moving forward. Keep in mind some protocols have multiple versions like Arbitrum One and Arbitrum Nova. Maybe we can clone the domain naming scheme from `dss-teleport`.

Reply:
This can be a good idea as well.
Ok, let's for now just remove the values from the config file, we can take the final decision on how to set it up later.